### ISSUE TITLE
Click to focus broken

### ISSUE THREAD

**Issue by alexozer** [baskerville/bspwm#494]
After pulling recently, I realized clicking windows to focus them (other than chromium) does not work; bspc commands to change focus do work though. I've also found that sometimes when a window is indicated to be focused with a border, it really is not.

I performed a bisect and found commit 12edac86fa3be32f799bedca4c8f5c0287a735b0 appears to be the culprit.

**Comment by alexozer** [comment]
Oops, missed the `click_to_focus` option.

**Comment by Lesik** [comment]
I am running 0.9.1 and the configuration `click_to_focus` does not seem to exist. The commit @alexozer mentioned happened after the 0.9.1 release. Nevertheless focusing windows by click does not do anything. Am I missing something or should I wait until 0.9.2? Why add a feature to a release but not its configuration toggle?

**Comment by alexozer** [comment]
@Lesik 
```
bspc config click_to_focus		true
bspc config pointer_modifier mod1
```

**Comment by Lesik** [comment]
@alexozer Thanks, but no, that doesn't work unfortunately.
I also `man bspc` and `/click` doesn't reveal any results, so I presume the configuration has not been added yet when the 0.9.1 release was made.

**Comment by MarcosCosmos** [comment]
The 0.9.1 release is about 7 months old and isn't particularly close to representing the latest version - click_to_focus was broken for me as well when I was working under the 0.9.1 release through the official package. I fixed it by replacing it with bspwm-git from the AUR. There are also a few other feature improvements that haven't been shipped to the package in the official repos (e.g. recepticle nodes).

**Comment by Lesik** [comment]
Hmm. Building from git is not really recommended (who knows what experimental code @baskerville is currently working on ;)) and many distros will not ship packages with software built from master. It would be great if @baskerville could release 0.9.2 so that non-Arch users get them new features (and the fix to this quite annoying `click_to_focus` thing).

**Comment by johan-bjareholt** [comment]
Click to focus is now broken for me on 0.9.2 from the arch repos.

Strange that a point release both changed the click to focus behavior as well as changed from using labels to identify monitors to hex numbers.

I was able to work around the hex numbers, but does anyone know how to fix click to focus?
I checked that it is indeed set to true with "bspc config click_to_focus"

Other pointer configs like pointer_follows_monitor seems to work though.
